src/DIRAC/Interfaces/API/Job.py

```diff
@@ -463,23 +463,6 @@ def setPlatform(self, platform):
 
         return S_OK()
 
-    #############################################################################
-    def setSubmitPool(self, backend):
```

Comment:
did you check that Ganga wasn't using it ?

Reply:
They're not 😃